tests/typecheck/managers/querysets/test_values_list.yml

```diff
@@ -26,6 +26,25 @@
                     name = models.CharField(max_length=100)
                     age = models.IntegerField()
 
+-   case: values_list_field_repetition
+    main: |
+        from myapp.models import MyUser
+
+        # Values tuples can have the same field repeated
+        values_tuple = MyUser.objects.values_list('name', 'name').get()
+        reveal_type(values_tuple)  # N: Revealed type is "Tuple[builtins.str, builtins.str]"
```

Comment:
And let's use `age` here as well to be sure that this is not just a single type that repeats itself :)

Reply:
Done - good suggestion!. I've doubled-checked that without the fix the new version of the test does fail.